infra/base-images/base-runner/coverage

```diff
@@ -111,21 +111,35 @@ function run_fuzz_target {
   fi
 }
 
+export GOPATH=$OUT/root/go
```

Comment:
as we install Go in `base-runner` itself, I don't think we need to re-define `GOPATH` here. Am I wrong?

Reply:
if we need to, let's do `GOPATH=$OUT/$GOPATH` to avoid hardcoding

Reply:
> as we install Go in base-runner itself, I don't think we need to re-define GOPATH here. Am I wrong?

I am afraid you are.
There are indeed 2 GOPATHs in base-runner :
- the system one (/root/go), which is used by gocovmerge
- the fuzz targets one (/root/go/root/go), which is used by running the targets

Do you see a better way to do this ?

Reply:
Is it important to use `/root/go/root/go` for running fuzz targets?